packages/xo-web/src/xo-app/logs/backup-ng/index.js

```diff
@@ -79,13 +88,18 @@ const COLUMNS = [
   },
   {
     name: _('jobStatus'),
-    itemRenderer: log => <LogStatus log={log} />,
+    itemRenderer: (log, { jobs }) => (
+      <LogStatus
+        disabled={get(() => jobs[log.jobId].type) !== 'backup'}
```

Comment:
Why not use `isEmpty(log.tasks)`?

Reply:
Right :+1:

Reply:
But it cannot works if an error throws at the job level.